pcdsdevices/lodcm.py

```diff
@@ -84,13 +158,129 @@ class LODCM(BaseInterface, Device):
 
     mono_line : str, optional
         Name of the mono, double-bounce beamline.
-    """
 
+    z1_prefix : str
+        LOM Xtal1 Z motor prefix.
+
+    x1_prefix : str
+        LOM Xtal1 X motor prefix.
+
+    y1_prefix : str
+        LOM Xtal1 Y motor prefix.
+
+    th1_prefix : str
+        LOM Xtal1 Theta motor prefix.
+
+    ch1_prefix : str
+        LOM Xtal1 Chi motor prefix.
+
+    h1n_m_prefix : str
+        LOM Xtal1 Hn motor prefix.
+
+    h1p_prefix : str
+        LOM Xtal1 Hp motor prefix.
+
+    z2_prefix : str
+        LOM Xtal2 Z motor prefix.
+
+    x2_prefix : str
+        LOM Xtal2 X motor prefix.
+
+    y2_prefix : str
+        LOM Xtal2 Y motor prefix.
+
+    th2_prefix : str
+        LOM Xtal2 Theta motor prefix.
+
+    ch2_prefix : str
+        LOM Xtal2 Chi motor prefix.
+
+    h2n_prefix : str
+        LOM Xtal2 Hn motor prefix.
+
+    diode2_prefix : str
+        LOM Xtal2 PIPS motor prefix.
+
+    dh_prefix : str
+        LOM Dia H motor prefix.
+
+    dv_prefix : str
+        LOM Dia V motor prefix.
+
+    dr_prefix : str
+        LOM Dia Theta motor prefix.
+
+    df_prefix : str
+        LOM Dia Filter Wheel motor prefix.
+
+    dd_prefix : str
+        LOM Dia PIPS motor prefix.
+
+    yag_zoom_prefix : str
+        LOM Zoom motor prefix.
+    """
     h1n = Cpt(H1N, ':H1N', kind='hinted')
```

Comment:
I think `h1n`'s name could be changed to something like `h1n_state` or positioner.... this guy is different from the other `h1n_m = UCpt(IMS, kind='normal', doc='LOM Xtal1 Hn')` - i think... i need to look into it more

Reply:
It's confusing because the `h1n` does include a `motor` subdevice... with hopefully the same PV as the `h1n_m`. These state devices have the motor embedded into them.

Reply:
In the old code we have something like this:

```
self._stateh1n = stateiocDevice("%s:H1N" % pvBase)

(full pv: XPP:LODCM:H1N)
```
and also:
```
self.h1n = h1n

(full pv: XPP:MON:MMS:09)
```

I just have `h1n_m` instead of `h1n` because it was taking, that's why I was thinking about changing the current `h1n` to something like `h1n_state`

Reply:
So from a programming perspective, I still have to have two components though correct?
```
h1n_state = Cpt(H1N, .....)
h1n = Cpt(IMS, ....)

h2n_state = Cpt(H2N, ....)
h2n = Cpt(IMS, ...)

... and so on...
```
They have different PVs...

Reply:
`InOutRecordPositioner` already automatically has a `motor` subdevice, but if it makes the implementation/usage easier it is good to duplicate it anyway.

I think using `_state` to designate the state devices is a good idea, it is very clear.